src/components/filters/RangeFilter.vue

```diff
@@ -118,10 +118,13 @@ export default Vue.extend({
       return Math.min(Math.max(value, min), max)
     },
     handleRangeValueChange () {
-      if (this.rangeValue[0] != null && this.rangeValue[0] !== '') {
+      if (this.rangeValue[0] === '') this.rangeValue[0] = null
```

Comment:
suggestion: always place conditional code in block, i.e. use 
`if ( foo ) {
  x = y
}`

instead of 
`if (foo) x = y`